app/data/repos.js

```diff
@@ -0,0 +1,10 @@
+const EleventyFetch = require('@11ty/eleventy-fetch')
+
+module.exports = async function () {
+  const url = 'https://api.github.com/orgs/x-govuk/repos'
+
+  return EleventyFetch(url, {
+    duration: '1d',
+    type: 'json'
+  })
+}
```

Comment:
This is clever! I wonder if just manually maintaining a list in JSON might be more flexible though? I guess we can see how it goes as other projects get added...

Reply:
Where's the fun it that! 😉

Was actually wondering if the list of resources should come from JSON data, actually…

Reply:
Ha, maybe?! I have a hunch that the Markdown file would be easier for people to edit to add new things?